homeassistant/components/google/config_flow.py

```diff
@@ -147,6 +145,9 @@ async def async_oauth_create_entry(self, data: dict) -> FlowResult:
         except ApiException as err:
             _LOGGER.debug("Error reading calendar primary calendar: %s", err)
```

Comment:
Maybe abort if this fails?

Reply:
Sure, removed gradeful degradation and now there is a how hard fail instead.